src/rdb/rdb_raft.c

```diff
@@ -2356,6 +2378,24 @@ rdb_raft_get_compact_thres(void)
 	return i;
 }
 
+static unsigned int
+rdb_raft_get_ae_max_entries(void)
+{
+	unsigned int n = 32;
+
+	d_getenv_int("RDB_AE_MAX_ENTRIES", &n);
```

Comment:
perhaps want to get the value of RDB_AE_MAX_ENTRIES and check if it is zero; if so, warn and instead use the default n=32 value? Otherwise I would expect some problems (unable to clone AE messages at all in rdb_raft_clone_ae() since the max # of entries would be zero).

Reply:
Done.